sandy/njoy.py

```diff
@@ -1043,14 +1091,27 @@ def _groupr_input(endfin, pendfin, gendfout, mat,
                                           spectrum_groupr[1::2]))
         text += [tab1]
         text += ["/"]
-    if mt is None:
+    if mt is None and kwargs["xs"]:
```

Comment:
I expected xs to be processed by default.
What if xs is not given?  There is no example about it